Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1298,13 +1298,14 @@ <h3>
 </p>
 <p>
     Um eine missbräuchliche mehrfache Verwendung der Aktualisierungsfunktion verhindern zu können,
-    wird die elektronische Signatur des aktualisierten digitalen COVID-Zertifikats in Form einer
-    speziellen Einwegverschlüsselung (sog. Hash-Wert) vorübergehend auf dem Serversystem der App
+    werden die elektronischen Signaturen des aktualisierten und des neuen digitalen
```

Comment:
This is even more confusing than the first version:
In the first version for us it's not clear why the hash of the updated DCC is stored, instead of the hash of the DCC to be updated. Now, the text handles an updated and additionally a new DCC?

Reply:
In my understanding: aktualisiertes Zertifikat = neues Zertifikat

Reply:
Yes, and I don't see a good reason to introduce a redundancy here.
My gut feeling is, that legal department has problems to understand what is the subject actually about. Maybe there was a talk with them that it should be "certificate to be updated" instead of "updated certificate", but they didn't understand correctly and added a "new certificate"? 🤔

Reply:
I think I have read somewhere that there are two cases that could explain this naming. In some cases extending a certificate is handled as "UPDATE" and wrongly issued certificates will create "NEW" certificates. Even if this is right, this would noch explain the reason for the confusion about the hashing

Reply:
Or maybe a possible PCR test would create a new recovery certificate.

(these are all just wild guesses 😅)